bayesian_testing/experiments/delta_normal.py

```diff
@@ -0,0 +1,298 @@
+from numbers import Number
+from typing import List, Tuple
+import numpy as np
+from bayesian_testing.experiments.base import BaseDataTest
+from bayesian_testing.metrics import eval_delta_normal_agg
+from bayesian_testing.utilities import get_logger
+
+logger = get_logger("bayesian_testing")
+
+
+class DeltaNormalDataTest(BaseDataTest):
+    """
+    Class for Bayesian A/B test for Delta-Normal data (Normal with non-positive values).
+    Delta-normal data is typical case of net profit data where sessions could have 0 values,
+    positive or negative values.
+    To handle this data, the evaluation methods are combining binary bayes model
+    for zero vs non-zero “conversion” and normal model.
+
+    After class initialization, use add_variant methods to insert variant data.
+    Then to get results of the test, use for instance `evaluate` method.
+    """
+
+    def __init__(self) -> None:
+        """
+        Initialize DeltaNormalDataTest class.
+        """
+        super().__init__()
+
+    @property
+    def totals(self):
+        return [self.data[k]["totals"] for k in self.data]
+
+    @property
+    def positives(self):
+        return [self.data[k]["positives"] for k in self.data]
```

Comment:
I am not sure if "positives" is a good wording. In case of delta-lognormal it was really the case. But here will be zeros vs normal-non-zeros. So maybe `non_zeros`?